### PR TITLE
Add support for primary and suggested shelf IDs

### PR DESCRIPTION
**Pull request by ziadoz**
- Added methods `addSuggestedPrimaryShelfIds()` and `addSuggestedShelfIds()` to add new shelf IDs.
- Updated item rendering to include shelf IDs.

### CODE REVIEW

File: src/Item.php [hunk 1]
```diff
@@ -504,6 +514,72 @@ public function setItemLogistics(ItemLogistics $itemLogistics)
...
+    private function setShelfId($type, $shelfIds)
+    {
+        if (! is_array($shelfIds)) {
+            $shelfIds = [$shelfIds];
+        }
+
+        $shelfIds = array_filter($shelfIds);
+        $shelfIds = array_unique($shelfIds);
+
+        $this->shelfIds[$type] = array_merge($this->shelfIds[$type], $shelfIds);
```
Comment (gabriel403) [inline src/Item.php]: Do we want to array_unique here too?

File: src/Item.php [hunk 2]
```diff
@@ -504,6 +514,72 @@ public function setItemLogistics(ItemLogistics $itemLogistics)
...
+    /**
+     * Set the shelf IDs for a type of shelf (primary or suggested).
+     * Note: Strips out any duplicate or empty values.
+     *
+     * @param $type
+     * @param $shelfIds
+     */
+    private function setShelfId($type, $shelfIds)
+    {
+        if (! is_array($shelfIds)) {
```
Comment (gabriel403) [inline src/Item.php]: I know we do this a bit, but is it better to always ensure an array comes in than to try and deal with it not being so?

File: src/Item.php (hunk 1 above)
Comment (ziadoz) [inline src/Item.php]: Yeah, good spot. Sorted.

File: src/Item.php (hunk 2 above)
Comment (ziadoz) [inline src/Item.php]: Yeah, I agree. Might be best done as a separate change though.

Comment (gabriel403) [inline src/Item.php]: It's a new private function, makes sense to do it now, then it's not something we have to cleanup later

Comment (ziadoz) [inline src/Item.php]: Done.

**Comment by gabriel403** [comment]
:+1:

**Pull request merged**